src/Microsoft.Extensions.Logging.Testing/AssemblyTestLog.cs

```diff
@@ -21,7 +21,7 @@ public class AssemblyTestLog : IDisposable
     {
         public static readonly string OutputDirectoryEnvironmentVariableName = "ASPNETCORE_TEST_LOG_DIR";
         private static readonly string LogFileExtension = ".log";
-        private static readonly int MaxPathLength = 245;
```

Comment:
Hmmm maybe make this configurable via an env var.

Reply:
That could be good